src/services/study/geo-data.ts

```diff
@@ -7,8 +7,13 @@
 
 import { backendFetchJson, getQueryParamsList } from '../utils';
 import { getStudyUrlWithNodeUuid } from './index';
+import { UUID } from 'crypto';
 
-export function fetchSubstationPositions(studyUuid, currentNodeUuid, substationsIds) {
+export function fetchSubstationPositions(
+    studyUuid: UUID,
+    currentNodeUuid: UUID | undefined,
```

Comment:
i think it better to use '?' to make this argument optional
```suggestion
    currentNodeUuid?: UUID,
```

Reply:
Done